fist/server.c

```diff
@@ -180,11 +254,14 @@ int start_server(struct config *config) {
 
     command_tree = NULL;
     // not a self balancing tree, be mindful of the order
+    bst_insert(&command_tree, "HELP", do_help);
```

Comment:
I think that the order should be

* HELP
* EXIT
* INDEX
* DROP
* KEYS
* DELETE
* SEARCH
* VERSION

(not that it matters too much)

Although maybe some time soon we should just add a bst_balance, or make the tree self balancing.